CONTRIBUTING.md

```diff
@@ -313,38 +317,38 @@ For example.
 ```go
 // config holds options for all animals.
 type config struct {
-	Weight      float64
-	Color       string
-	MaxAltitude float64
+    Weight      float64
```

Comment:
In some places `\t` was used for indentation in others there were 4 spaces.

I decided to go with 4 spaces everywhere because my markdown linter does not complain when spaces are used 😉

Reply:
Can we stick with hard tabs for the Go code? That way when people copy it it will be validly formatted code.

As for markdownlint we just merged #1842 which should configure your IDE to ignore those hard tabs when this is updated to the main branch.

Reply:
Sure. I will fix it today.

Reply:
Changed here and in other places as well: https://github.com/open-telemetry/opentelemetry-go/pull/1841/commits/1bc2d83cf2bd44ba8b9f1a4344aeb555f47f9f26